src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25224.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	internal class Issue25224 : _IssuesUITest
```

Comment:
Would be nice to include the sample from the test here https://github.com/dotnet/maui/tree/main/src/Controls/samples/Controls.Sample/Pages/Controls/CollectionViewGalleries/EmptyViewGalleries
In this case, we will have samples to validate all the EmptyView scenarios (using a Label, View, Template, TemplateSelector).

Reply:
@jsuarezruiz , 
Thanks! I’ve added the tests to EmptyViewGalleries for all EmptyView scenarios as requested. Let me know if there’s anything else needed!